### ISSUE TITLE
PS2 GSM - upscale HD input - passthrough problem

### ISSUE THREAD

**Issue by ste23droid** [ramapcsx2/gbs-control#520]
Hi there,
I have setup my PAL PS2 with GBS control using component cables.
Everything working, except one thing I cannot understand.

PS2 is able to output video with component cables up to 1080i using GSM software.

Anyway, every time I set GSM options on my PS2, GBS Control goes in pass through automatically and I cannot use filters to smooth more the image.

e.g PS2 GSM setup to output 720p 60hz. GBS control setup for output 1920x1080 60hz.
Output on tv becomes 720p 60hz, ignoring my GBS control initial setup. It goes on passthrough automatically.

Instead, I would like PS2 to output with GSM any resolution (possibly 720p), but still have the GBS Control output 1080p.
The ideal setup should be the one allowing to use GSM and then further enhance with GSB control.

Is this possible?
How can it be achieved?

Have a nice day

**Comment by invaderlex** [comment]
no, You can't have 720p source and 1080p output. You can only upscale 240p, 480i and 480p.
